.azure-pipelines/azure-pipelines-win.yml

```diff
@@ -0,0 +1,105 @@
+# This file was generated automatically from conda-smithy. To update this configuration,
+# update the conda-forge.yml and/or the recipe/meta.yaml.
+# -*- mode: yaml -*-
+
+jobs:
+- job: win
+  pool:
+    vmImage: vs2017-win2016
+  timeoutInMinutes: 360
+  strategy:
+    maxParallel: 4
+    matrix:
```

Comment:
Is there a reason for not having Python 3.8/3.9/3.10 here? I thought they used to be built too.